app/views/candidate_interface/course_choices/have_you_chosen.html.erb

```diff
@@ -0,0 +1,19 @@
+<% content_for :title, page_title('have_you_chosen') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
```

Comment:
Thinking maybe this should go to `candidate_interface_course_choices_index_path` and potentially the others as well.

Reply:
Done!